crates/test_utils/wasm/wasm_workspace/create_entry/src/coordinator.rs

```diff
@@ -18,6 +18,28 @@ fn msg() -> Msg {
     Msg("hello".into())
 }
 
+#[hdk_extern]
+pub fn block_agent(target: AgentPubKey) -> ExternResult<()> {
+    HDK.with(|h| {
+        h.borrow().block_agent(holochain_zome_types::block::BlockAgentInput {
```

Comment:
Is there a preference between `HDK.with` and the associated function that hides grabbing the current HDK instance?

Reply:
@ThetaSinner the latter is just a sugar wrapper and isn't even comprehensive re: all the things you can do using `with`